src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -217,8 +206,6 @@ public void writeToNBT(NBTTagCompound compound)
         NBTTagCompound workManagerCompound = new NBTTagCompound();
         workManager.writeToNBT(workManagerCompound);
         compound.setTag(TAG_WORK, workManagerCompound);
-
-        //compound.setInteger(TAG_AUTO_HOSTILE, autoHostile);
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)